neqo-transport/src/cc/classic_cc.rs

```diff
@@ -161,17 +161,27 @@ impl<T: WindowAdjustment> CongestionControl for ClassicCongestionControl<T> {
     }
 
     // Multi-packet version of OnPacketAckedCC
-    fn on_packets_acked(&mut self, acked_pkts: &[SentPacket], min_rtt: Duration, now: Instant) {
+    fn on_packets_acked(
+        &mut self,
+        acked_pkts: &[SentPacket],
+        min_rtt: Duration,
+        latest_rtt: Duration,
+        smooth_rtt: Duration,
```

Comment:
Any reason why this can't be passed `&RttEstimate`?  The function is called from `Path::on_packets_acked()`, so I think that is much easier.  Then your logging calls might just invoke the `Debug` implementation on `RttEstimate` (which might need to be improved, as it uses the default).

Reply:
Your tests can probably build a `const RTT_ESTIMATE`.

Reply:
Thanks. This is neat. 
I'll do this.